packages/rpc/src/index.ts

```diff
@@ -62,89 +62,83 @@ export const Code = component('rpc' as Component, {
   Unauthorized: '' as StatusCode,
   Forbidden: '' as StatusCode,
   BadRequest: '' as StatusCode,
-  UnknownMethod: '' as StatusCode<{method: string}>
+  UnknownMethod: '' as StatusCode<{ method: string }>
 })
 
 export interface RequestStream {
-  send: <T extends any[]>(req: Request<T>) => void
+  send: (request: Request<any>) => void
 }
 
 interface Operation {
-  id: number
-  request: () => void // Perform request again
+  id: ReqId
   promise: Promise<any>
-  resolve: <T> (value: T) => void
+  resolve: <T>(value: T) => void
+  reject: (reason?: any) => void
```

Comment:
Is not it `reason: error`?

Reply:
Nope, 
<img width="957" alt="Screenshot 2021-06-09 at 11 00 51" src="https://user-images.githubusercontent.com/477235/121291183-08d98700-c912-11eb-99a1-46226458d782.png">